public/notes.js

```diff
@@ -0,0 +1,88 @@
+import {
+    inputEnabled,
+    setDiv,
+    message,
+    setToken,
+    token,
+    enableInput,
+  } from "./index.js";
+  import { showLoginRegister } from "./loginRegister.js";
+  import { showAddEdit } from "./addEdit.js";
+  
+  let notesDiv = null;
+  let notesTable = null;
+  let notesTableHeader = null;
+  
+  export const handleNotes = () => {
+    notesDiv = document.getElementById("notes");
+    const logoff = document.getElementById("logoff");
+    const addNote = document.getElementById("add-note");
+    notesTable = document.getElementById("notes-table");
+    notesTableHeader = document.getElementById("notes-table-header");
+  
+    notesDiv.addEventListener("click", (e) => {
+      if (inputEnabled && e.target.nodeName === "BUTTON") {
+        if (e.target === addNote) {
+          showAddEdit(null);
+        } else if (e.target === logoff) {
+            setToken(null);
+            message.textContent = "You have been logged off.";
+            notesTable.replaceChildren([notesTableHeader]);
+
+            showLoginRegister();
+        } else if (e.target.classList.contains("editButton")) {
+            message.textContent = "";
+            showAddEdit(e.target.dataset.id);
+        } else if (e.target.classList.contains("deleteButton")) {
+          message.textContent = "";
+          showAddEdit(e.target.dataset.id);
```

Comment:
When we click the delete button, the application behaves as if we had clicked the edit button. Therefore, we need to create a method that calls the endpoint to delete the note and replace the current action of calling the showAddEdit method with this new method.

Reply:
Hello Jose
I have managed to add a function to delete
Kindly confirm if I have done it right.